app/src/main/res/layout/item_collectible_collectible.xml

```diff
@@ -12,6 +12,7 @@
         android:layout_width="match_parent"
```

Comment:
Why is it required to wrap this card view in a constraint layout?

Reply:
we want to have a white background for the whole width of the item